roles/serviceassurance/tasks/component_elasticsearch.yml

```diff
@@ -15,9 +15,9 @@
               memory: 4Gi
             requests:
               memory: 1Gi
-        redundancyPolicy: ZeroRedundancy
+        redundancyPolicy: {{ "FullRedundancy" if haEnabled else "ZeroRedundancy" }}
```

Comment:
Given the current state, I think the haEnabled is not necessary; simply testing esNodeCount > 1 would have the same effect. Perhaps this is meant for future use? I'm trying to consider whether any of my work would benefit from such a flag, or whether similarly setting a "size" or "nodecount" property to >1 is all I really need. We can feel this out as we make progress.

Reply:
I was considering whether this was needed as well. Potentially, we can provide reasonable defaults for HA (esNodeCount, etc), and have them set all the time, but ignored unless haEnabled==True. This could be useful.

Reply:
You actually want to check on this not just via a single flag either. You need > 2 nodes for FullRedundancy I'm pretty sure. If you have `elasticsearchNodeCount: 2` then you want this to be `SingleRedundancy` I think.

Reply:
Ah yes, if we need 3 nodes that's important to know. I'm not sure if there has been an testing on failed ES states yet or not.

Reply:
These docs [1] say nothing about limitations for nodeCount vs FullRedundancy. For nodeCount==2, SingleRedundancy is equivalent to SingleRedundancy AFAICT. 
The only limitation I saw noted was SingleRedundancy doesn't work on single node deployment.

I tested nodeCount=2 with FullRedundancy, and the smoketests ran. 

[1] https://docs.openshift.com/container-platform/4.1/logging/config/efk-logging-elasticsearch.html

Reply:
Did you test what happens to the stream of events when you kill a node?

Reply:
No, no I did not. I just see the smoke tests passing and the pods recovering.